src/main/webapp/pages/userSignin.html

```diff
@@ -20,11 +20,20 @@
 <body>
     <div class="container">
         <div class="row">
-            <h1 class="signin-title">Sign In</h1>
+            <h1 class="signin-title">Log In</h1>
         </div>
         <div class="row">
-            <input class="name-input" placeholder="Name Here"/>
+            <input class="name-input" id="login-user-name" />
+            <label class="label">Name</label>
+
+            <!-- messages -->
+            <p class="error hide" id="error-msg">Please use letters and numbers only.</p>
+            <p class="error hide" id="empty-msg">Username cannot be empty.</p>
+            <p class="success hide" id="success-msg">Log In Successful!</p>
```

Comment:
Can't you use a single p-tag and modify the message based on the status? Why have3 separate elements and show/hide each one?
Also can you name the class as error-hide or errorHide etc, without space